src/App.tsx

```diff
@@ -18,6 +18,8 @@ export default function App(): JSX.Element {
   return (
     <div className={classes.appContainer}>
       <MobxRouter store={store} />
+      <NetWorth />
```

Comment:
why are you using the components here?

Reply:
Sorry, I put there for visualize the component and forgot to delete. Gonna do that now.